crates/polars-core/src/serde/mod.rs

```diff
@@ -103,6 +114,36 @@ mod test {
         assert!(df.frame_equal_missing(&out));
     }
 
+    #[test]
+    #[cfg(feature = "dtype-array")]
+    fn test_serde_array_owned_json() {
+        let arrays = vec![
+            UInt8Chunked::new("test", vec![1u8, 2, 3, 4, 5]),
+            UInt8Chunked::new("test2", vec![2u8, 3, 4, 5, 6]),
+        ];
+        let mut builder = get_fixed_size_list_builder(&DataType::UInt8, 2, 5, "array").unwrap();
```

Comment:
Could you rewrite this test without calling `get_fixed_size_list_builder` and especially without `unsafe` code?

Reply:
@c-peters an `Array/FixedSizeList` stores all values in a single buffer.

So you can create a `UInt8Array([8, 2, 3, 4, 5, 2, 3, 4, 5, 6])` and store that in the `FixedSizeListArray` (probably under `values`) with the `width` parameter. 


That arrow `FixedSizeListArray` can then be converted into a `Series` with `Series::try_from(("name", array))`.